openage/convert/main.py

```diff
@@ -20,6 +22,10 @@
     "drive_c/Program Files/Microsoft Games/Age of Empires II/"
 _STANDARD_PATH_IN_64BIT_WINEPREFIX =\
     "drive_c/Program Files (x86)/Microsoft Games/Age of Empires II/"
+_REGISTRY_KEY = \
+    "HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Microsoft Games\\"
+_REGISTRY_SUFFIX_AOK = "Age of Empires\\2.0"
+_REGISTRY_SUFFIX_TC = "Age of Empires II: The Conquerors Expansion\\1.0"
```

Comment:
will be done. I just thought of these as implementation details, thus made them "protected"